config/routes.rb

```diff
@@ -13,14 +13,18 @@
 
   resources :profiles
 
+  # likes
   post 'like/:id', to: 'tweets#like', as: :like
-  get 'likes', to: 'tweets#like_index', as: :all_users
+  get 'like/:id', to: 'tweets#like_index'
 
-  # resources :likes
-  get "tweets/route", to: 'tweets#route'
+  # visitor counts
+  get 'visitor/:id', to: 'tweets#visitors', as: :visitor
```

Comment:
Move this to `resources :tweets` block.

Reply:
`get 'visitor/:id'`
`'like/:id`
ABove both routes are not as per rails conventions, please refer routes section again and update both